DS13/code/modules/borg/items+structures.dm

```diff
@@ -29,6 +29,18 @@
 		qdel(x)
 	return TRUE
 
+/obj/item/organ/body_egg/borgNanites
+	name = "nanite cluster"
+	desc = "A metal lattice..every part of it moves and swims at its own will."
+	zone = BODY_ZONE_CHEST
+	slot = "borg_infection"
+	icon_state = "heart-c-on"
+	color = "#008000"
+
+/obj/item/organ/body_egg/borgNanites/Remove(mob/living/carbon/M, special = 0) //Allows for deborging
```

Comment:
What happens if I take it out and insert it into someone else because I am evil and or dumb and or kmc